dask/dataframe/core.py

```diff
@@ -1491,7 +1491,7 @@ def _reduction_agg(self, name, axis=None, skipna=True, split_every=False, out=No
                 split_every=split_every,
             )
             if isinstance(self, DataFrame):
-                result.divisions = (min(self.columns), max(self.columns))
+                result.divisions = (self.columns.min(), self.columns.max())
```

Comment:
Are these changes from your empty reduction PR? Or are they necessary here? Ideally we keep them separate so that we don't introduce merge conflicts.

Reply:
They were originally here but I decided to move them to a seperate PR since they are distinct changes. I can remove these changes after merging #5804